packages/jellyfish-api-core/src/category/masternode.ts

```diff
@@ -140,6 +140,22 @@ export class Masternode {
     return await this.client.call('setgov', [input], 'number')
   }
 
+  /**
+   * Set special governance variables with activation height specified
+   *
+   * @param {Record<string, any>} input json object
+   * @param {number} activationHeight
+   * @return {Promise<string>} hash
+   *
+   */
+  async setGovWithHeight (input: Record<string, any>, activationHeight: number): Promise<string> {
```

Comment:
```suggestion
  async setGovHeight (input: Record<string, any>, activationHeight: number): Promise<string> {
```

Like what you wrote in your md file